lib/box/cfg.go

```diff
@@ -178,6 +182,19 @@ func (vars *EnvVars) Environ() (environ []string) {
 	return environ
 }
 
+// WriteTo writes the environment variables to the specified writer w.
+// WriteTo implements io.WriterTo
+func (vars EnvVars) WriteTo(w io.Writer) (n int64, err error) {
+	cw := &countingWriter{}
+	for _, v := range vars {
+		// quote value as it may contain spaces
+		if _, err := fmt.Fprintf(io.MultiWriter(w, cw), "%v=%q\n", v.Name, v.Val); err != nil {
```

Comment:
Doesn't `fmt.Fprintf` return number of bytes written? Why is a separate "counting writer" needed?